### ISSUE TITLE
Pasting half of 🔆 into a text field crashes neos.

### ISSUE THREAD

**Issue by Psychpsyo** [Neos-Metaverse/NeosPublic#2213]
## Describe the bug
When pasting only the first character that makes up the 🔆 symbol into a text field, Neos hangs forever and must be force quit.

## Relevant issues
Didn't find any.

## To Reproduce
1. Copy this: 🔆
2. Paste it into a text field in Neos.
3. Move your cursor two to the left, placing it in front of the sun.
4. Shift + Right Arrow to select only the first part of the sun.
5. Ctrl+C, Ctrl+V
6. Neos is now frozen.

## Expected behavior
Neos should not freeze and just paste the symbol, showing either it's representation in the font or the missing character rectangle.

## Bug information (please complete the following information):
 - How often does it happen: Always
 - Does the bug persist after restarting Neos? Yes
 - Neos Version: 2021.5.
 - Neos Platform: Windows, probably any.
 - Regression: Don't know.

## Reporters:
Cameron#7818

**Comment by 3x1t-5tyl3** [comment]
Tried this: Windows prompted me with a "Try to recover" I tried that and recovered and got all this in my log. After neos froze.
`18:49:01.664 ( 90 FPS)	Exception in RunningCoroutine: FrooxEngine.TextEditor+<EditCoroutine>d__72
System.IO.IOException: Unable to read data from the transport connection: An established connection was aborted by the software in your host machine.
. ---> System.Net.Sockets.SocketException: An established connection was aborted by the software in your host machine.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer, System.Int32 offset, System.Int32 size, System.Net.Sockets.SocketFlags socketFlags) [0x00016] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
  at System.Net.Sockets.NetworkStream.Read (System.Byte[] buffer, System.Int32 offset, System.Int32 size) [0x0009b] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
   --- End of inner exception stack trace ---
  at System.Net.Sockets.NetworkStream.Read (System.Byte[] buffer, System.Int32 offset, System.Int32 size) [0x000e2] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
  at System.IO.Stream.ReadByte () [0x00007] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.IO.BinaryReader.ReadByte () [0x0000d] in <9577ac7a62ef43179789031239ba8798>:0 
  at BaseX.BinaryReaderExtensions.Read7BitEncoded (System.IO.BinaryReader br) [0x00005] in <7065cb282c784500b91cd90d70653d65>:0 
  at BaseX.BinaryReaderExtensions.ReadEnumBinary[E] (System.IO.BinaryReader br) [0x0001c] in <7065cb282c784500b91cd90d70653d65>:0 
  at NeosmonClient.ClipboardData..ctor (System.IO.BinaryReader reader) [0x00006] in <e09ab5f6fbb1467585d0ef04e1234326>:0 
  at NeosmonClient.Neosmon.GetClipboardData () [0x0001d] in <e09ab5f6fbb1467585d0ef04e1234326>:0 
  at WindowsPlatformConnector.EnsureLatestClipboardData () [0x0001b] in <7131999581ac41649931d05e85950984>:0 
  at WindowsPlatformConnector.get_ContainsText () [0x00000] in <7131999581ac41649931d05e85950984>:0 
  at FrooxEngine.TextEditor+<EditCoroutine>d__72.MoveNext () [0x00151] in <082dfa144d5b4721a90ea99f0cc9d83a>:0 
  at FrooxEngine.CoroutineManager.RunStep (FrooxEngine.CoroutineHandle coroutine) [0x0001d] in <082dfa144d5b4721a90ea99f0cc9d83a>:0 

   at void BaseX.UniLog.Error(string message, bool stackTrace)
   at void FrooxEngine.DebugManager.Error(object obj, bool stackTrace)
   at void FrooxEngine.CoroutineManager.RunStep(CoroutineHandle coroutine)
   at int FrooxEngine.CoroutineManager.ExecuteWorldQueue(double deltaTime)
   at bool FrooxEngine.World.RefreshStep()
   at bool FrooxEngine.World.Refresh()
   at void FrooxEngine.WorldManager.UpdateStep(double maxMilliseconds)
   at bool FrooxEngine.WorldManager.RunUpdateLoop(double maxMilliseconds)
   at void FrooxEngine.Engine.UpdateStep(double maxMilliseconds)
   at void FrooxEngine.Engine.RunUpdateLoop(double maxMilliseconds)
   at void FrooxEngineRunner.Update()
18:49:16.889 ( 90 FPS)	Exception in RunningCoroutine: FrooxEngine.TextEditor+<EditCoroutine>d__72
System.IO.IOException: Unable to write data to the transport connection: An existing connection was forcibly closed by the remote host.
. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host.

  at System.Net.Sockets.Socket.Send (System.Byte[] buffer, System.Int32 offset, System.Int32 size, System.Net.Sockets.SocketFlags socketFlags) [0x00016] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
  at System.Net.Sockets.NetworkStream.Write (System.Byte[] buffer, System.Int32 offset, System.Int32 size) [0x0009b] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
   --- End of inner exception stack trace ---
  at System.Net.Sockets.NetworkStream.Write (System.Byte[] buffer, System.Int32 offset, System.Int32 size) [0x000e2] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
  at System.IO.Stream.WriteByte (System.Byte value) [0x0000b] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.IO.BinaryWriter.Write (System.Byte value) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at BaseX.BinaryWriterExtensions.Write7BitEncoded (System.IO.BinaryWriter bw, System.UInt64 integer) [0x0001a] in <7065cb282c784500b91cd90d70653d65>:0 
  at BaseX.BinaryWriterExtensions.WriteEnumBinary[E] (System.IO.BinaryWriter bw, E value) [0x0002e] in <7065cb282c784500b91cd90d70653d65>:0 
  at NeosmonClient.Neosmon.GetClipboardData () [0x00011] in <e09ab5f6fbb1467585d0ef04e1234326>:0 
  at WindowsPlatformConnector.EnsureLatestClipboardData () [0x0001b] in <7131999581ac41649931d05e85950984>:0 
  at WindowsPlatformConnector.get_ContainsText () [0x00000] in <7131999581ac41649931d05e85950984>:0 
  at FrooxEngine.TextEditor+<EditCoroutine>d__72.MoveNext () [0x00151] in <082dfa144d5b4721a90ea99f0cc9d83a>:0 
  at FrooxEngine.CoroutineManager.RunStep (FrooxEngine.CoroutineHandle coroutine) [0x0001d] in <082dfa144d5b4721a90ea99f0cc9d83a>:0 

   at void BaseX.UniLog.Error(string message, bool stackTrace)
   at void FrooxEngine.DebugManager.Error(object obj, bool stackTrace)
   at void FrooxEngine.CoroutineManager.RunStep(CoroutineHandle coroutine)
   at int FrooxEngine.CoroutineManager.ExecuteWorldQueue(double deltaTime)
   at bool FrooxEngine.World.RefreshStep()
   at bool FrooxEngine.World.Refresh()
   at void FrooxEngine.WorldManager.UpdateStep(double maxMilliseconds)
   at bool FrooxEngine.WorldManager.RunUpdateLoop(double maxMilliseconds)
   at void FrooxEngine.Engine.UpdateStep(double maxMilliseconds)
   at void FrooxEngine.Engine.RunUpdateLoop(double maxMilliseconds)
   at void FrooxEngineRunner.Update()
18:49:27.556 ( 89 FPS)	Exception in RunningCoroutine: FrooxEngine.TextEditor+<EditCoroutine>d__72
System.IO.IOException: Unable to write data to the transport connection: An existing connection was forcibly closed by the remote host.
. ---> System.Net.Sockets.SocketException: An existing connection was forcibly closed by the remote host.

  at System.Net.Sockets.Socket.Send (System.Byte[] buffer, System.Int32 offset, System.Int32 size, System.Net.Sockets.SocketFlags socketFlags) [0x00016] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
  at System.Net.Sockets.NetworkStream.Write (System.Byte[] buffer, System.Int32 offset, System.Int32 size) [0x0009b] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
   --- End of inner exception stack trace ---
  at System.Net.Sockets.NetworkStream.Write (System.Byte[] buffer, System.Int32 offset, System.Int32 size) [0x000e2] in <ef151b6abb5d474cb2c1cb8906a8b5a4>:0 
  at System.IO.Stream.WriteByte (System.Byte value) [0x0000b] in <9577ac7a62ef43179789031239ba8798>:0 
  at System.IO.BinaryWriter.Write (System.Byte value) [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at BaseX.BinaryWriterExtensions.Write7BitEncoded (System.IO.BinaryWriter bw, System.UInt64 integer) [0x0001a] in <7065cb282c784500b91cd90d70653d65>:0 
  at BaseX.BinaryWriterExtensions.WriteEnumBinary[E] (System.IO.BinaryWriter bw, E value) [0x0002e] in <7065cb282c784500b91cd90d70653d65>:0 
  at NeosmonClient.Neosmon.GetClipboardData () [0x00011] in <e09ab5f6fbb1467585d0ef04e1234326>:0 
  at WindowsPlatformConnector.EnsureLatestClipboardData () [0x0001b] in <7131999581ac41649931d05e85950984>:0 
  at WindowsPlatformConnector.get_ContainsText () [0x00000] in <7131999581ac41649931d05e85950984>:0 
  at FrooxEngine.TextEditor+<EditCoroutine>d__72.MoveNext () [0x00151] in <082dfa144d5b4721a90ea99f0cc9d83a>:0 
  at FrooxEngine.CoroutineManager.RunStep (FrooxEngine.CoroutineHandle coroutine) [0x0001d] in <082dfa144d5b4721a90ea99f0cc9d83a>:0 

   at void BaseX.UniLog.Error(string message, bool stackTrace)
   at void FrooxEngine.DebugManager.Error(object obj, bool stackTrace)
   at void FrooxEngine.CoroutineManager.RunStep(CoroutineHandle coroutine)
   at int FrooxEngine.CoroutineManager.ExecuteWorldQueue(double deltaTime)
   at bool FrooxEngine.World.RefreshStep()
   at bool FrooxEngine.World.Refresh()
   at void FrooxEngine.WorldManager.UpdateStep(double maxMilliseconds)
   at bool FrooxEngine.WorldManager.RunUpdateLoop(double maxMilliseconds)
   at void FrooxEngine.Engine.UpdateStep(double maxMilliseconds)
   at void FrooxEngine.Engine.RunUpdateLoop(double maxMilliseconds)
   at void FrooxEngineRunner.Update()`
